docs/cpp/configure-intellisense.md

```diff
@@ -0,0 +1,121 @@
+
+- "IntelliSense: Ready" = IntelliSense has been configured for the C/C++ Extension and will automatically activate if you interact with the editor, for example, by writing code.
+- "IntelliSense: Updating" = IntelliSense is actively working to determine any code completions, syntax highlighting, etc. based on changes you're making to your code
+- "IntelliSense: Not configured" = IntelliSense isn't configured, and will not provide your with code completion features. To configure, navigate you to the [configuration quickpickpick](/docs/cpp/configure-intellisense#_selecting-a-compiler-for-IntelliSense.md) as explained below.
+
+![language-status-bar](images/intellisense/language-status-bar.png)
+
+### Option 3: Check the `c_cpp_properties.json` file
+
+All configuration settings are stored in your project's `c_cpp_properties.json` file. Navigate to this file by selecting **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`) in VS Code. Check the "IntelliSense mode" to understand your configuration. To learn more about the `c_cpp_properties.json` file, see the [c_cpp_properties.json schema reference](/docs/cpp/c-cpp-properties-schema-reference.md).
+
+![Command Palette](images/cpp/command-palette.png)
+
+## How to configure IntelliSense
+
+If you have determined that IntelliSense isn't configured, you have multiple options to configure:
+
+### Option 1. Select a configuration option through the configuration Quick Pick
+
+Open the quick pick by entering **Select IntelliSense Configuration** in the Command Palette (`kb(workbench.action.showCommands)`), which shows you a dropdown with all of the configuration options found by the C/C++ Extension on your machine.
+
+![Compiler Quick Pick](images/intellisense/compiler-quick-pick.png)
+
+Select one of the options available. You can return to the Configuration Quick Pick at any point to change which option is being configured for IntelliSense.
+
+If no options are available in the quick pick, there are multiple options to help you install a compiler which you can use for IntelliSense. On a Windows machine, select the “Help me install a compiler” option, which redirects you to the step-by-step walkthrough of how to install a C/C++ compiler. On a Mac or Linux machine, select "Install a compiler" and navigate through the prompts to have a compiler either the clang or GCC compiler installed on your machine.
+
+### Option 2. Edit your IntelliSense configurations
+
+Open your IntelliSense configuration by selecting **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`).
+
+For configuring a compile_commands.json file, enter the full path of the compile_commands.json file into the Compile commands field. For example, if your compile_commands.json file is in the root of your workspace, use `${workspaceFolder}/compile_commands.json`.
```

Comment:
Use inline code block formatting for filenames so "For configuring a `compile_commands.json` file..."